swarm/api/http/server.go

```diff
@@ -967,24 +950,24 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 	// check the root chunk exists by retrieving the file's size
 	if _, err := reader.Size(ctx, nil); err != nil {
 		getFileNotFound.Inc(1)
-		Respond(w, r, fmt.Sprintf("file not found %s: %s", r.uri, err), http.StatusNotFound)
+		RespondError(w, r, fmt.Sprintf("file not found %s: %s", uri, err), http.StatusNotFound)
 		sp.Finish()
 		return
 	}
 
 	buf, err := ioutil.ReadAll(newBufferedReadSeeker(reader, getFileBufferSize))
```

Comment:
Make sure to note that ReadAll should not be released as production code.

Reply:
I meant to fix this over the last few days, but didn't get around it. Let's remove it as part of this PR. Sorry about that.